wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallDegradationViewController/UIAlertController+DegradedCall.swift

```diff
@@ -77,7 +77,8 @@ extension UIAlertController {
 
     static func degradedMLSConference(
```

Comment:
suggestion: add the verb make (or build) to this func to make clear on the caller side that this is more than a simple property getter.